### PR TITLE
Update license text

### LINKED ISSUE #148: Make license UI less confusing
# Consider the current license UI

## Entitled (selected by default)

image

## Evaluation

image

# IBM requests this UI be changed to the following.

I will use pseudojson to describe the desired UI. I understand the assignee must rework this to achieve the desired "show/hide" effect depending on **Entitled** or **Evaluation** being selected.

In the following pseudojson, any element without a "text" property should be unchanged from its existing code.

## Entitled

```json
{
    "name": "ibmIdInfo",
    "type": "Microsoft.Common.InfoBox"
},

{
    "name": "ibmUserId",
    "type": "Microsoft.Common.TextBox"
},

{
    "name": "ibmUserPwd",
    "type": "Microsoft.Common.PasswordBox"
},

{
    "name": "licenseInfo",
    "type": "Microsoft.Common.InfoBox",
    "options": {
        "icon": "Info",
        "text": "Accept the <a href=\"https://ibm.biz/tWASBaseLicenseAzureVMs\" target=\"_blank\">IBM WebSphere Application Server License Agreement</a> to proceed with the deployment."
    }
}
```

In other words, the **Entitled** UI shows an **InfoBox**, followed by the IBMid UI entry fields, followed by another **InfoBox**.

## Evaluation

```json
{
    "name": "ibmIdInfo",
    "type": "Microsoft.Common.InfoBox",
    "text": "By accepting the IBM License Agreement you are accepting the Evaluation terms of the license. An evaluation deployment does not include WebSphere interim fixes (iFixes). The evaluation period runs for 60 days after which you must purchase entitlement to continue to use WebSphere."
},

{
    "name": "licenseInfo",
    "type": "Microsoft.Common.InfoBox",
    "options": {
        "icon": "Info",
        "text": "Accept the <a href=\"https://ibm.biz/tWASBaseLicenseAzureVMs\" target=\"_blank\">IBM WebSphere Application Server License Agreement</a> to proceed with the deployment."
    }
}

```

In other words, the **Evaluation** UI shows an **InfoBox** immediately followed by another **InfoBox**.

### PR DESCRIPTION
**Pull request by majguo**
The PR is to resolve #148 .

Signed-off-by: Jianguo Ma <jiangma@microsoft.com>

### CODE REVIEW

**Review by edburns — APPROVED** [review]

File: src/main/arm/createUiDefinition.json [hunk 1]
```diff
@@ -63,7 +62,7 @@
                 "type": "Microsoft.Common.InfoBox",
                 "options": {
                     "icon": "Info",
-                    "text": "[concat(if(bool(basics('useTrial')), 'By accepting the IBM License Agreement you are accepting the Evaluation terms of the license. An evaluation deployment does not include WebSphere interim fixes (iFixes). The evaluation period runs for 60 days after which you must purchase entitlement to continue to use WebSphere.', 'Accept the IBM License Agreement to proceed with the deployment.'), ' <a href=\"https://ibm.biz/tWASBaseLicenseAzureVMs\" target=\"_blank\">IBM WebSphere Application Server</a>.')]"
+                    "text": "Accept the <a href='https://ibm.biz/tWASBaseLicenseAzureVMs' target='_blank'>IBM WebSphere Application Server License Agreement</a> to proceed with the deployment."
```
Comment (git4rk) [inline src/main/arm/createUiDefinition.json:65]: License URL is for Base. It should use ND license URL.

Comment (edburns) [reply src/main/arm/createUiDefinition.json:65]: I wondered about that.  Can you please reply with the correct ibm.biz URL, @git4rk ?

Comment (git4rk) [reply src/main/arm/createUiDefinition.json:65]: @edburns its https://ibm.biz/tWASNDLicenseAzureVMs

Comment (edburns) [reply src/main/arm/createUiDefinition.json:65]: Thanks, I will push another commit, @git4rk .

**Comment by edburns** [comment]
@git4rk  updated with another commit.  Ready to merge.  As soon as you do, I'll kick off the publish.

**Review by git4rk — APPROVED** [review]

**Pull request merged**